src/combined.py

```diff
@@ -7,14 +9,13 @@
 from src.newsplease import NewsPleaseParser
 from src.readability import ReadabilityParser
 from src.base import HTMLParser, ParsedHTML
-from src.config import MIN_NO_LINES_FOR_VALID_TEXT
+from src.config import MIN_NO_LINES_FOR_VALID_TEXT, HTTP_REQUEST_TIMEOUT
 
 logger = logging.getLogger(__name__)
 
 
 class CombinedParser(HTMLParser):
-    """
-    Runs the NewsPlease parser on the given URL. If any paragraph is longer than `max_paragraph_words` or the NewsPlease output is empty, it falls back to the Readability parser.
+    """Runs the NewsPlease parser on the given URL. If any paragraph is longer than `max_paragraph_words` or the NewsPlease output is empty, it falls back to the Readability parser.
```

Comment:
we do need to sort out our docstring style. Obviously it's not important at the moment, but for future generated documentation, PEP257 is really nice.

Reply:
should be using pep257 atm  - that docstring just doesn't have any `arguments` or `returns sections` - unless i've missed something?